### PR TITLE
feat: 支持使用的远端资源的管理, issue: #6579

### PR DESCRIPTION
**Pull request by tming**
支持使用的远端资源的管理

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-ci?pullRequest=6580) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-ci?pullRequest=6580) before we can accept your contribution.<br/><hr/>**tomtian(田明)** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-ci?pullRequest=6580) it.</sub>

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-ci?pullRequest=6580) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-ci?pullRequest=6580) before we can accept your contribution.<br/><hr/>**tomtian(田明)** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-ci?pullRequest=6580) it.</sub>

File: src/backend/booster/bk_dist/booster/pkg/booster.go [hunk 1]
```diff
@@ -457,7 +459,10 @@ func (b *Booster) runWithApply(pCtx context.Context) (int, error) {
 
 		case e := <-event:
 			if err = b.parseEvent(e); err != nil {
-				return b.runDegradeWorks(ctx)
+				// TODO (tomtian) : do not degrade when failed to apply resource
+				// return b.runDegradeWorks(ctx)
+				blog.Infof("booster: failed to apply resource for work(%s) with error:%v", b.workID, err)
+				return b.runWorks(ctx, nil, nil)
```
[the remark this answers is not in the document]
Comment (tming) [reply src/backend/booster/bk_dist/booster/pkg/booster.go:465]: 这儿只打印资源申请失败的日志，但不转本地，因为新流程会在后续继续自动尝试申请资源

**Comment by tming** [comment]
BK-CI GITHUB PR CHECK@PULL_REQUEST    检查失败，关掉后重新打开，再次触发该检查

**Review by flyy1012 — APPROVED** [review]
ok

**Pull request merged**